### ISSUE TITLE
Cannot verify my own session

### ISSUE THREAD

**Issue by kuba-orlik** [vector-im/riot-web#12821]
### Description

I've set up my own Riot-web instance and upon login I've been presented with a "verify your session" popup:

![2020-03-21_16-03](https://user-images.githubusercontent.com/2697916/77229438-dcf3a980-6b85-11ea-8f7a-a3b18e990e88.png)

After clicking "verify" a new modal appeared:

![2020-03-21_16-04](https://user-images.githubusercontent.com/2697916/77229443-eaa92f00-6b85-11ea-927e-febf081536e1.png)

Then, after clicking "continue", a new modal showed up:

![2020-03-21_16-04_1](https://user-images.githubusercontent.com/2697916/77229448-f98fe180-6b85-11ea-84da-822a8836c9c2.png)

The spinner on that one just kept spinning forever.

I got confused - why is it waiting for my confirmation? How do I confirm that?

Logs being sent: yes

### Version information

- **Platform**: web (in-browser)

For the web app:

- **Browser**: Firefox 74
- **OS**: Manjaro Linux
- **URL**: private server, riot-web version: 1.5.13

**Comment by tleydxdy** [comment]
I also had this problem, just setup my own copy, and I have cross-signing enabled. All other devices gets automatically trusted, but it doesn't trust itself.

**Comment by bwindels** [comment]
@kuba-orlik Thanks for reporting. You see this notification when you have other sessions that haven't been cross-signed yet. It's waiting for you to accept the verification request on your other session in firefox with the given device id. Did you open that other session to see if the request popped up there?

The UX is indeed still a bit confusing here, you can track progress on this at https://github.com/vector-im/riot-web/issues/12725

**Comment by bwindels** [comment]
> I also had this problem, just setup my own copy, and I have cross-signing enabled. All other devices gets automatically trusted, but it doesn't trust itself.

This sounds like a different issue, can you please file a separate issue for this?

**Comment by kuba-orlik** [comment]
> Does this clear it up?

yes, indeed. Upon further inspection I realized that right after registration i have TWO sessions created in my browser. One active, and other one is invisible. I've removed the invisible session and now it works properly. But other users on my instance encountered the same issue and it's very confusing for them! So I guess the issue is about two sessions being created. Should I edit this issue or create a new one?
